docs/pages/build-reference/ios-capabilities.md

```diff
@@ -93,6 +93,36 @@ You can run `EXPO_DEBUG=1 eas build` to get more detailed logs regarding the cap
 
 If you have trouble using this feature, you can disable it with the environment variable `EXPO_NO_CAPABILITY_SYNC=1`.
 
-To see all of the currently enabled capabilities, visit [Apple Developer Portal][apple-dev-portal], and find the bundle identifier matching your app, if you click on it you should see a list of all the currently enabled capabilities.
+To see all of the currently enabled capabilities, visit [Apple Developer Console][apple-dev-console], and find the bundle identifier matching your app, if you click on it you should see a list of all the currently enabled capabilities.
 
-[apple-dev-portal]: https://developer.apple.com/account/resources/identifiers/list
+## Manual setup
+
+There are two ways to manually enable Apple capabilities, both systems will require any existing Apple provisioning profiles to be regenerated.
+
+### Xcode
+
+> Preferred method for bare workflow or projects that do **not** using [Expo Prebuild](/workflow/prebuild) to continuously generate the native `ios` and `android` directories.
+
+1. Open the `ios/` directory in Xcode with `xed ios`. If you don't have an `ios/` directory, run `npx expo prebuild -p ios` to generate one.
+2. Then follow [this guide][apple-enable-capability].
+
+### Apple Developer Console
+
+First step is to add the respective key/value pairs to your `ios/[app]/[app].entitlements` (or more specific entitlements file for multi-target apps). You can refer to [Supported Capabilities](#supported-capabilities) to determine which entitlements keys should be added.
+
+1. Log into [Apple Developer Console][apple-dev-console]. Go to "Certificates, Identifiers, & Profiles" > "Identifiers".
```

Comment:
```suggestion
1. Log into [Apple Developer Console][apple-dev-console]. Click on "Certificates, IDs & Profiles", then navigate to "Identifiers" page.
```

Wording nit. Also, can't we also use link in there? 
* https://developer.apple.com/account/resources/identifiers/list

Reply:
this is the only link I use for apple dev console, it's on `apple-dev-console`